src/smolagents/remote_executors.py

```diff
@@ -249,6 +340,93 @@ def cleanup(self):
         except Exception as e:
             self.logger.log_error(f"Error during cleanup: {e}")
 
+def _websocket_send_execute_request(code: str, ws) -> str:
```

Comment:
I think you introduced duplicate functions while resolving the conflicts with the main branch: this function was already defined and it is present in line 431.